content/millennium/r4/financial/support/coverage.md

```diff
@@ -197,6 +224,33 @@ X-Request-Id: ef7c0ee60a8cf431403fe82d9009640b
 
 <%= disclaimer %>
 
+### Example - Encounter-level Public Coverage
+
+#### Request
+
+    POST https://fhir-ehr-code.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/Coverage
+
+#### Body
+
+<%= json(:r4_coverage_encounter_social_healthcare_create) %>
+
+#### Response
+
+<%= headers status: 201 %>
+<pre class="terminal">
+Cache-Control: no-cache
+Content-Length: 0
+Content-Type: text/html
+Date: Tue, 22 Oct 2019 15:59:33 GMT
+Etag: W/"42-0"
+Last-Modified: Tue, 22 Oct 2019 15:59:30 GMT
+Location: https://fhir-ehr-code.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/Coverage/PH-589763-11500257-11500257
```

Comment:
nit: Could we change one of the '11500257' values to something else? I am assuming these each represent a patient/encounter id, and it may confuse somebody if they are the same value.

Reply:
https://github.com/cerner/fhir.cerner.com/pull/676/commits/1328e46531ce56c47bac659e645c62b93d99d79e